docs/framework/docker/aspnetmvc.md

```diff
@@ -227,15 +219,7 @@ To remove the container, issue a `docker rm` command:
 docker rm randomanswers
 ```
 
-## Summary
-
-In this topic, you've seen the steps to move and run an existing
-ASP.NET MVC application in a Windows Server container. Running an
-existing application does not require any changes to your
-application. You need to run the tasks to publish your application,
-build a Docker image, and start that image in a new container. Leveraging
-Windows Server Containers are the easiest path to migrate your existing
-applications to this environment.
+## Resources
```

Comment:
Those are link labels. They're not rendered there. They're referenced earlier in the topic. If you open your topic on stage, you'll see that.